src/configurations/nf/style/_overides.scss

```diff
@@ -12,3 +12,12 @@ $primary-action-color: #404B63;
   background-color: #125E81;
   background-blend-mode: multiply;
 }
+
+// Customized for nf.
+// Per design, the table should extend to full page width but there aren't enough columns to fill the empty space.
+.DetailsPage.tab-layout .QueryWrapperPlotNav{
+  table.grip-flex {
+    width: 100% !important;
```

Comment:
Is this a general change? (Can this be applied to all QueryWrapperPlotNav table.grip-flex ?)

Reply:
Yes, I moved the fix to SRC: https://github.com/Sage-Bionetworks/Synapse-React-Client/pull/765